### ISSUE TITLE
Scout explore without pause for hostile units.

### ISSUE THREAD

**Issue by edbouverie** [We-the-People-civ4col-mod/Mod#242]
This may exist as part of a wider option but I find that late game on large maps scouts are best left automated - the problem is that wild animals will pause their search and when on a huge map this can lead to having to move 10+ scouts away from the animal, waiting a turn then automating again. Is it possible to add an option remove scouts from this behaviour, perhaps in the game options tab?

**Comment by ErikSandbraaten** [comment]
Well, one way to accomplish this without changing the existing behaviour of AUTOMATE_EXPLORE would be to add a new AutomateType (perhaps AUTOMATE_EXPLORE_IGNORE_DANGER ?) which does not 
check CvUnitAI::AI_breakAutomation. Another alternative would be to re-use the existing COMMAND_IGNORE_DANGER (used by wagons) and have it apply to the scout unit.

**Comment by raystuttgart** [comment]
Let us please not mess with that, it is not worth the effort to add a game option.
And the current behaviour is most likely wanted by most players.

**Comment by raystuttgart** [comment]
Scouts should stop when they meet dangers.
The behaviour is as designed / wanted.

**Comment by MrZorG33** [comment]
I do not agree. scouts will sometimes stop automatic exploration of the area if there is an enemy in their line of sight (even if the player does not see it). but sometimes they attack them even when they are wounded (and therefore die).
similarly, automated ocean explorers pass by the enemy unless attacked. but I will write about this in a separate topic (it seems that there was one).

**Comment by raystuttgart** [comment]
So what now?

What is it that you want as behaviour?
Stop or not stop when hostiles are sighted.

This thread requests: Scouts should **not** stop.
You however request: Scount **should** stop if I understand you correctly.

-----

If A) "Scouts **should stop** when encountering hostiles" is the desired behaviour. -> This issue should be closed.
If B) "Scouts **should not stop** when encountering hostiles" is the desired behaviour. -> This issue should stay open.

-----

So what is it?
A) or B)

Everything else has little to do with this 3 year old issue.
